shared/constants/types/autoreset.tsx

```diff
@@ -1,8 +1,8 @@
-export type State = Readonly<{
```

Comment:
this is a bummer. Readonly<> is way slower than making everything readonly

Reply:
😢